brewchat/src/main/java/com/example/brewchat/ChatService.java

```diff
@@ -158,6 +164,31 @@ public void onError(List<String> errors) {
         });
     }
 
+    public void loadContacts() throws QBResponseException {
+        QBRoster roster = QBChatService.getInstance().getRoster();
+        ArrayList<Integer> userIds = new ArrayList<>(roster.getEntries().size());
+        for (QBRosterEntry user : roster.getEntries()) userIds.add(user.getUserId());
+        if (roster.getEntries().size() == 0)
+            EventBus.getDefault().post(new UsersLoadedEvent(new ArrayList<QBUser>()));
+
+        QBUsers.getUsersByIDs(userIds, new QBPagedRequestBuilder(userIds.size(), 1),
+                new QBEntityCallbackImpl<ArrayList<QBUser>>() {
+                    @Override
+                    public void onSuccess(ArrayList<QBUser> result, Bundle params) {
+                        super.onSuccess(result, params);
+                        EventBus.getDefault().post(new UsersLoadedEvent(result));
```

Comment:
Please consider to use `User` instead of `QBUser`.

I think that we should not spread `QB` classes across the application.